.github/workflows/dockerfile-check.yml

```diff
@@ -16,6 +16,7 @@ jobs:
           docker build -t forest-test .
           docker run --rm forest-test --version
       - name: Set WORKFLOW_URL
+        if: always()
```

Comment:
Given it's used only for fiascoes, should it be `failure()`?

Reply:
Yes, but if we test changes, `if: failure()` needs to be commented out, while `if: always()` always works

Reply:
I'm not really convinced, but it's a trivial job so not a big deal.